v1/dnscrypt-resolvers.csv

```diff
@@ -163,16 +173,20 @@ dnscry.pt-libertylake-ipv4,dnscry.pt-libertylake-ipv4,"DNSCry.pt Liberty Lake -
 dnscry.pt-libertylake-ipv6,dnscry.pt-libertylake-ipv6,"DNSCry.pt Liberty Lake - DNSCrypt, no filter, no logs, DNSSEC support (IPv6 server)",,,,2,yes,yes,no,[2602:fc24:18:33f2::ab1],2.dnscrypt-cert.dnscry.pt,B083:7AB6:5CAE:B3EF:4787:2618:B41F:BC21:590F:D959:0CA5:84C2:2DBE:A2A9:77D1:ADCD,
 dnscry.pt-lima-ipv4,dnscry.pt-lima-ipv4,"DNSCry.pt Lima - DNSCrypt, no filter, no logs, DNSSEC support (IPv4 server)",,,,2,yes,yes,no,161.132.47.185,2.dnscrypt-cert.dnscry.pt,1CEF:433E:24CB:7D15:68D2:FF63:DD75:B8BC:8FFA:4D40:8091:87E6:BC97:B8C9:4B3F:0775,
 dnscry.pt-lima-ipv6,dnscry.pt-lima-ipv6,"DNSCry.pt Lima - DNSCrypt, no filter, no logs, DNSSEC support (IPv6 server)",,,,2,yes,yes,no,[2001:470:1f2a:1de::2],2.dnscrypt-cert.dnscry.pt,1CEF:433E:24CB:7D15:68D2:FF63:DD75:B8BC:8FFA:4D40:8091:87E6:BC97:B8C9:4B3F:0775,
-dnscry.pt-london-ipv4,dnscry.pt-london-ipv4,"DNSCry.pt London - DNSCrypt, no filter, no logs, DNSSEC support (IPv4 server)",,,,2,yes,yes,no,178.239.174.244,2.dnscrypt-cert.dnscry.pt,8F66:DC44:BEBB:62C6:0CEA:2D99:2B92:5FFE:1CBE:FE09:ABB6:6140:8417:6BDB:F2CA:31EE,
+dnscry.pt-london-ipv4,dnscry.pt-london-ipv4,"DNSCry.pt London - DNSCrypt, no filter, no logs, DNSSEC support (IPv4 server)",,,,2,yes,yes,no,45.67.84.132,2.dnscrypt-cert.dnscry.pt,8F66:DC44:BEBB:62C6:0CEA:2D99:2B92:5FFE:1CBE:FE09:ABB6:6140:8417:6BDB:F2CA:31EE,
```

Comment:
IP address was out-of-date, but the host key is the same. IP address manually verified at https://www.dnscry.pt/public-resolvers/lon01